addons/sourcemod/scripting/scp_sf/scps/939.sp

```diff
@@ -66,7 +66,7 @@ public void SCP939_OnMaxHealth(int client, int &health)
 
 public void SCP939_OnSpeed(int client, float &speed)
 {
-	speed += (1.0-(Health[client]/HealthMax))*SpeedExtra;
+	speed += (1.0-(Pow(Health[client]/HealthMax), 2.0))*SpeedExtra;
```

Comment:
This would now make it quicker to get fast speed at first, then slowly gaining to max at low health, which I'm assuming that's intended.